tests/versions/tslib.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Validate the tslib and tsc version is consistent and corresponds to each other.
+ */
+import { execSync } from "child_process";
+import { readFileSync } from "fs";
+import JSON from "json5";
```

Comment:
Can we use [JSON.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse), and move comment from versions.json to the test code? This way we can avoid direct devDependency on `json5` package.

Another option is to use `_comment` key for comment as follows:
```json
{
  "_comment": "You need to make sure 'tslib' version is compatible with typescript version.",
  "tslib": "^2.3.0",
  "typescript": "~4.3.2"
}
```

Reply:
If you are worry of the dependency, JSON5 is already in our dependency graph. `"_comment"` does not draw as much attention as real comment either.